src/test/resources/zodiac_pro_check.config.yml

```diff
@@ -0,0 +1,92 @@
+$abstract: false
```

Comment:
I would not check in of these old configs.  This will be a maintenance issue and confuse future team members